processing/data_collection/gazette/spiders/pa_belem.py

```diff
@@ -0,0 +1,64 @@
+import scrapy
```

Comment:
Imports are not compliant with PEP8:
https://www.python.org/dev/peps/pep-0008/?#imports

It is not a rule in this project, but I believe that trying to follow PEP8 will make the code more beautiful :-)